server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -1537,6 +1538,7 @@ private void upgradeTsFileResourceCallBack(TsFileResource tsFileResource) {
           }
         }
       }
+      UpgradeSevice.getINSTANCE().stop();
```

Comment:
If we need to stop UpgradeSevice after upgrading complete, I think we should also stop it if there is no file to be upgraded.

Reply:
fixed